waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -468,9 +487,21 @@
                                             :task-stats {:running 2, :healthy 2, :unhealthy 0, :staged 0}})
                    {:active-instances
                     [(scheduler/make-ServiceInstance
+                       {:healthy? false
+                        :host "0.0.0.0"
+                        :id "test-app-1234.test-app-1234-abcd0-0"
+                        :k8s/pod-phase "Pending"
+                        :k8s/primary-container-state {:waiting "ContainerCreating"}
+                        :log-directory "/home/myself/r0"
+                        :port 8080
+                        :service-id "test-app-1234"
+                        :started-at (du/str-to-date "2014-09-13T00:24:46Z" k8s-timestamp-format)})
+                     (scheduler/make-ServiceInstance
                        {:healthy? true
                         :host "10.141.141.11"
                         :id "test-app-1234.test-app-1234-abcd1-0"
+                        :k8s/pod-phase "Running"
+                        :k8s/primary-container-state {:running "reason_unknown"}
```

Comment:
Again, is this really a helpful state? I'd rather have something like this:
```
:k8s/primary-container-state {:state :running}
```
and
```
:k8s/primary-container-state {:state :terminated
                              :reason "OOMKilled"}
 ```

Reply:
Changed and added a warning for when multiple states are provided by k8s.